### ISSUE TITLE
Circular Navigation Plugin fires an exception

### ISSUE THREAD

**Issue by danielalves** [tulios/jquery.silver_track#19]
We can see the exception log on any browser console:
```
jquery.silver_track.circular_navigator.js:231 Uncaught TypeError:
Cannot read property 'autoPlay' of nulljquery.silver_track.circular_navigator.js:231 $.silverTrackPlugin._turnOnListener
```

**Comment by tulios** [comment]
@MarcosOcf  could you check this issue?

**Comment by MarcosOcf** [comment]
@tulios  yes, i'm doing this.

**Comment by tulios** [comment]
fixed with pull request #23
